src/components/ProgressIndicator/ProgressIndicator.md

```diff
@@ -0,0 +1,16 @@
+### Notes for use
+
+`ProgressIndicator` spans the full width of its parent container. If you need to make it appear
+a certain width, you have to set the width property of its parent container to the desired
+width.
+
+If you need your ariaValueText translated, you'll have to do translation in the parent component and
+pass a translated string as `ProgressIndicator` will not be able to do it for you.
```

Comment:
This applies to any label. @swese44 should we add this notice everywhere, or remove it from here?

Reply:
ya i'd say probably just remove it

Reply:
Removed with: https://github.com/Microsoft/YamUI/pull/122/commits/ce18ac01a4b37fb2048e11575b1481a50e49c523